Files/powerGate/Modules/Communication.psm1

```diff
@@ -2,12 +2,43 @@
 Import-Module powerGate
 
 #TODO: configure the powerGate server url and port
-$powerGateServerName = $ENV:Computername
+$powerGateServerName = getRelatedPGServerName
 $powerGateServerPort = "8080"
+# define here witch Vault is used for test and witch one for production
+$testVaults = @(
+	"TestVault1",
```

Comment:
This template should work out of the box, therefore a default Vault form Autodesk is named "Vault". This should be configured here.